web/src/upload.js

```diff
@@ -185,16 +176,36 @@ export async function upload_files(uppy, config, files) {
                 get_item("textarea", config),
             );
             compose_ui.autosize_textarea(get_item("textarea", config));
-            uppy.addFile({
+            file.id = uppy.addFile({
                 source: get_item("source", config),
                 name: file.name,
                 type: file.type,
                 data: file,
             });
         } catch {
             // Errors are handled by info-visible and upload-error event callbacks.
-            break;
+            continue;
```

Comment:
Using `continue` instead of `break` because we can show multiple banners -- previously we'd show the first error then stop trying anything.